app/src/main/java/org/fossasia/openevent/general/auth/GoogleAuthBuilder.kt

```diff
@@ -0,0 +1,110 @@
+package org.fossasia.openevent.general.auth
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.content.IntentSender
+import android.os.Build
+import android.os.Bundle
+import android.support.v4.app.ActivityCompat
+import android.support.v4.app.ActivityCompat.startActivityForResult
+import android.support.v4.app.ActivityCompat.startIntentSenderForResult
+import android.support.v4.app.FragmentActivity
+import android.support.v4.content.ContextCompat.startActivity
+import android.widget.EditText
+import com.google.android.gms.auth.api.Auth
+import com.google.android.gms.auth.api.credentials.Credential
+import com.google.android.gms.auth.api.credentials.CredentialRequest
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.api.CommonStatusCodes
+import com.google.android.gms.common.api.GoogleApiClient
+import com.google.android.gms.common.api.Status
+import org.fossasia.openevent.general.MainActivity
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.utils.nullToEmpty
+import timber.log.Timber
+
+class GoogleAuthBuilder(private val context: Context, private val activity: FragmentActivity): GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
```

Comment:
This is not a builder

Reply:
Thanks for all the recommended changes :+1: